### PR TITLE
Adds AI core screen selection to character preferences.

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
tweak: AI core display screen can now be set in character preferences. 
fix: AI core display screen will now be restore when revived.
spellcheck: Corrected some inconsistent capitalization in the player preferences screen.
imageadd: Readded some forgotten AI sprites.
fix: Fixed Hades AI death animation not playing.
/:cl:

I got tired of having to manually select this every single round ever.

### CODE REVIEW

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -40,6 +40,117 @@ GLOBAL_LIST_EMPTY(caps_list)
...
+
+/proc/resolve_ai_icon(input)
+	if(!input)
+		return "ai"
+	else
+		if(input == "Random")
+			input = pick(GLOB.ai_core_display_screens - "Random")
+		switch(input)
+			if(":thinking:")
+				return "ai-:thinking:"
```
Comment (Jordie0608) [inline code/_globalvars/lists/flavor_misc.dm]: If you rename a few of the iconstates you could collapse this entire switch into just `return "ai-[ckey(input)]"`

**Comment by ShizCalev** [comment]
Added support for death transitions since I noticed Hades had one that was unused. I'll see about getting a few more done in the future.
https://i.imgur.com/jeS1CWN.gifv

**Comment by AnturK** [comment]
Should fall back to default if the preference is outdated. Looks good otherwise.

**Comment by ShizCalev** [comment]
Antur; Already does.

**Pull request merged**